### PR TITLE
My Home: update styling for Block-Editor card and add analytics

### PR DESCRIPTION
**Pull request by gwwar**
This PR updates:
*  Styling for "Master the Block Editor" to better match existing cards
* Adds some missing analytics events to the card
* Enhances the InlineSupportLink to accept tracks and stats analytics

- [x] Before landing drop the update in primary for `<MasteringGutenberg />`. This was added only for easier testing/reviewing.

| Before  | After |
| ------------- | ------------- |
|  78721460-81644280-791f-11ea-9e82-0d510b209245 | Screen Shot 2020-04-15 at 4 12 41 PM |

| Open Article  |
| ------------- |
| Screen Shot 2020-04-15 at 4 02 15 PM |

### Testing Instructions
* Start the branch
* Visit http://calypso.localhost:3000 (My Home section) 
* In console type `localStorage.debug='calypso:analytics*'`
* Click on customizing post and pages. Verify that the support article opens in the appropriate area to display the right video. 
* Verify that the following analytics events fire
Screen Shot 2020-04-15 at 3 56 52 PM
* Click on Adjusting Settings of Blocks. Verify that the support article opens in the appropriate area to display the right video. 
* Verify that the following analytics events fire:
Screen Shot 2020-04-15 at 3 57 10 PM

* Verify no regressions in InlineSupportLink
* For example try clicking on the podcast InlineSupportLink under Settings > Writing
* The Support article should appear as expected
Screen Shot 2020-04-15 at 4 01 24 PM
* Redux events fired by InlineSupportLink should look like the following:

| Open Article without Analytics  | Open Article With Analytics |
| ------------- | ------------- |
| Screen Shot 2020-04-15 at 4 06 14 PM | Screen Shot 2020-04-15 at 4 06 32 PM |

### CODE REVIEW

File: client/my-sites/customer-home/locations/primary/index.jsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ const Primary = ( { checklistMode, cards } ) => {
 	}
 	return (
 		<div className="primary">
+			<MasteringGutenberg />
```
Comment (gwwar) [inline client/my-sites/customer-home/locations/primary/index.jsx:28]: Do not let me commit this hard-coded line!

File: client/my-sites/customer-home/cards/education/mastering-gutenberg/index.jsx [hunk 2]
```diff
@@ -49,6 +49,14 @@ const MasteringGutenberg = () => {
 					showIcon={ false }
 					text={ translate( 'Adjusting settings of blocks' ) }
 				/>
+				<InlineSupportLink
+					supportPostId={ 145498 }
+					supportLink={ localizeUrl( 'https://support.wordpress.com/free-photo-library' ) }
+					showIcon={ false }
+					text={ translate(
+						'Add free beautiful copyright-free photos to create stunning designs'
```
Comment (a8ci18n) [inline client/my-sites/customer-home/cards/education/mastering-gutenberg/index.jsx:57]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'The WordPress.com Free Photo Library integrates your site with beautiful copyright-free photos to create stunning designs.' )` *ES Score: 7*

**Comment by danhauk** [comment]
I pushed up some changes to clean up the styles here.

Screen Shot 2020-04-09 at 3 40 33 PM

Something like this could work for sure, but I'll think a bit more the pros and cons on a combined approach like this vs. some kind of pager and separate cards.

**Comment by gwwar** [comment]
I can spin up the alternate with the pager and we can see what folks think feels better. 

Unfortunately the dot pager is tied to the guide component https://wordpress.github.io/gutenberg/?path=/story/components-guide--default so I'll need to temporarily duplicate this in Calypso, while adding in an upstream PR to Gutenberg to break that out for re-use

**Comment by gwwar** [comment]
> I pushed up some changes to clean up the styles here.

Feel free to push directly to the branch too @danhauk

**Comment by gwwar** [comment]
After thinking about this a bit, let's go with a pager for now. We have two new block-editor videos, so I'll keep this WIP PR to handle that update.

**Comment by gwwar** [comment]
@danhauk should we use a https://wpcalypso.wordpress.com/devdocs/design/vertical-nav here for consistency? I notice we're using this in other cards. (Though the grow card does hide the borders)

**Comment by gwwar** [comment]
@automattic/manage @Aurorum this one should be ready for review now 👍

File: client/components/inline-support-link/README.md [hunk 3]
```diff
@@ -27,3 +27,7 @@ render() {
 - `showText` - *optional* (bool) Whether or not to display the link text. Default `true`.
 - `showIcon` - *optional* (bool) Whether or not to display the "help-outline" Gridicon. Default `true`.
 - `iconSize` - *optional* (number) Gridicon size. Default `14`.
+- `tracksEvent` - *optional* (string) Tracks Analytics Event name
```
Comment (mmtr) [inline client/components/inline-support-link/README.md:30]: Thanks for adding analytics support to this! ❤️

**Pull request merged**